simtools/util/names.py

```diff
@@ -911,7 +911,7 @@ def simtelLogFileName(run, primary, arrayName, site, zenith, azimuth, label=None
 
     Warning
     -------
-        zst extension is hardcoded here.
+        zst extension is hard coded here.
```

Comment:
One word is actually OK (in particular in this context).

Reply:
OK - reversed that. Although technically wrong, one would probably search the code for hardcoded.